src/Symfony/Component/Cache/Adapter/FilesystemAdapter.php

```diff
@@ -20,10 +20,16 @@ class FilesystemAdapter extends AbstractAdapter
 {
     private $directory;
 
-    public function __construct($directory, $defaultLifetime = null)
+    public function __construct($directory, $defaultLifetime = 0, $namespace = '')
     {
         parent::__construct('', $defaultLifetime);
```

Comment:
shouldn't the `$namespace` be also passed as the first argument to the parent call?